www/source/demo/windows/steps/10.html.slim

```diff
@@ -0,0 +1,24 @@
+section
+ h1 Next Steps
+
+ p Now that you have completed the web application guide, here are some additional resources to help you:
+
+ p <b>Documentation</b>
+
+ ul
+  li #{link_to 'Writing packages','/docs/developing-packages#sharing-pkgs'}
+  li #{link_to 'Plan syntax guide','/docs/reference'}
+  li #{link_to 'Habitat Infographics','/docs/diagrams'}
+  li #{link_to 'Using Habitat','/docs/using-habitat/'}
+
+ p <b>Windows Specific Learning</b>
+
+ ul
+  li #{link_to 'Windows Blog Posts','/blog/windows/'}
+  li #{link_to 'Running Habitat as a Windows Service','https://github.com/habitat-sh/windows-service'}
+  li #{link_to 'Windows Plans', 'https://github.com/habitat-sh/core-plans/search?q=filename%3Aplan.ps1&unscoped_q=filename%3Aplan.ps1'}
```

Comment:
Seems like it'd be nice to be able to do this in Builder!

Reply:
yeah it would be nice but is it possible?

Reply:
ah I see. you are envisioning a better future!